core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/SampleCVSlide.java

```diff
@@ -0,0 +1,21 @@
+package uk.ac.bbsrc.tgac.miso.core.data;
+
+public interface SampleCVSlide extends SampleTissueProcessing {
+
+  public static final String SAMPLE_CLASS_NAME = "CV Slide";
+
+  Integer getCuts();
```

Comment:
We need to speak with Ilinca to confirm the actual requirements, as you and I got different information from her.